README.md

```diff
@@ -2,167 +2,353 @@
+  --exists              does a file in a directory exist (default: False)
+  --single COUNT        minimum number of single comments (default: None)
+  --multiple COUNT      minimum number of multi comments (default: None)
+  --language {Java,Python}
+                        language for the single comments (default: None)
+  --paragraphs COUNT    minimum number of paragraphs (default: None)
+  --words WORDS         minimum number of words in paragraphs (default: None)
+  --command COMMAND     command to run (default: None)
+  --executes            does a command execute without error (default: False)
+  --fragment FRAGMENT   fragment that exists in code or output (default: None)
+  --count COUNT         how many of an entity should exist (default: None)
+  --exact               equals instead of a minimum number (default: False)
+
+```
 
-Indicates to look for a number of single-line comments. This should be followed
-by a number of single-line comments to look for. The language must also be
-specified with the `--langauge` option.
+GatorGrader employs many checks to ensure that you configure it with the correct
+command-line arguments. For instance, if you type the command `pipenv run
+python3 gatorgrader.py --command ls --fragment pytest --count 1 --exacts` (which
+spells the argument as `--exacts` instead of `--exact`), then you will receive
+the following error message `gatorgrader.py: error: unrecognized arguments:
+--exacts`. While a command like  `pipenv run python3 gatorgrader.py --fragment
+pytest --count 1 --exact` uses the correct names for all of the specified
+arguments, it does not tell GatorGrader what type of deliverable to check (e.g.,
+a program's source code or a command's output), thus leading GatorGrader to
+report `Incorrect command-line arguments`.
+
+Since GatorGrader's commands support many different checks for the various
+deliverables of a project in a computer science course, this documentation will
+walk through several example commands. Using these examples, you are encouraged
+to try out new combinations of the arguments. If you notice any problems with
+using GatorGrader, the tool's developers ask that you raise an issue on the
+issue tracker. To start, the following command uses GatorGrader to ensure that
+the `internal/java` directory contains the file called `Sample.java` and the
+subsequent commands check that this file contains at least two single-line
+comments (e.g., those lines that start with `//`) and two multiple-line comments
+(e.g., content that is surrounded by `/** */`). In these and the remaining
+examples, you will first see the command and then the output that it produces.
+Finally, while in practice you would normally type these commands on a single
+line in your terminal window, we give them on multiple lines to ensure that
+they are easily viewed on a wide variety of displays.
 
-- `--multicomments`
+```
+pipenv run python3 gatorgrader.py \
```

Comment:
Command line entry should be preceded by `$` in blocks where both input and output are shown, by convention. Is there reasoning behind not following this convention here?

Reply:
Agreed, @Michionlion, I will add that now. Thanks for pointing out this issue! Do I need the `$` only on the first line for the command? And, is my approach to multiple lines okay? Let me know and I will be glad to fix these problems right away!